packages/botonic-react/src/util/dom.js

```diff
@@ -26,9 +29,17 @@ export const scrollToBottom = ({ timeout = 200, behavior = 'smooth' } = {}) => {
   }
 }
 
-export const getWebchatElement = () =>
-  document.getElementById('botonic-webchat')
+export const getWebchatElement = host =>
+  host && host.querySelector('#botonic-webchat')
```

Comment:
Again here, as @vanbasten17 said, I think it would be nice to have all this ids and classes as constants.